installer/templates/phx_umbrella/mix.exs

```diff
@@ -25,6 +25,7 @@ defmodule <%= @root_app_module %>.MixProject do
   # and cannot be accessed from applications inside the apps/ folder.
   defp deps do<%= if @html and Version.match?(System.version(), ">= 1.13.4") do %>
     [
+      <%= @phoenix_dep_umbrella_root %>,
```

Comment:
Should this dep be included in test only? It's not required in a regular app, it's here just to override local path.

Reply:
Do we use the `--dev` flag for tests? If so, we should include it whenever the `--dev` flag is given. :)